proposals/0002-web-rtc.md

```
@@ -0,0 +1,457 @@
+- Tests: common and platform-specific test stubs
+
+Engine factories
+
+- [JS/Wasm](#create-a-client): object `JsWebRtc` : `WebRtcClientEngineFactory<JsWebRtcEngineConfig>`
+    - Uses browser `RTCPeerConnection` and `Navigator` media devices
+- [Android](#create-a-client): object `AndroidWebRtc` : `WebRtcClientEngineFactory<AndroidWebRtcEngineConfig>`
+    - Requires Android context; uses `PeerConnectionFactory` and Android media devices
+    - If you supply a custom `MediaTrackFactory`, also set `AndroidWebRtcEngineConfig.rtcFactory`; otherwise engine
+      initialization will fail
+- [iOS](#create-a-client): object `IosWebRtc` : `WebRtcClientEngineFactory<IosWebRtcEngineConfig>`
+    - If you supply a custom `MediaTrackFactory`, also set `IosWebRtcEngineConfig.rtcFactory`; otherwise engine
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)